package.json

```diff
@@ -155,7 +155,7 @@
     "jest-environment-jsdom": "29.6.4",
     "js-yaml": "4.1.0",
     "mammoth": "1.6.0",
-    "marked": "4.3.0",
+    "marked": "9.0.0",
     "marked-terminal": "5.2.0",
```

Comment:
The 'marked' package has been updated from version 4.3.0 to 9.0.0. Please ensure that this major version upgrade does not introduce breaking changes in the application.